dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -236,6 +234,8 @@
  */
 public class ESContentletAPIImpl implements ContentletAPI {
 
+    private static Lazy<Boolean> ENABLED_UNIQUE_FIELDS_DATA_BASE_VALIDATION = Lazy.of(() ->
```

Comment:
Let's do `DATABASE` instead of `DATA_BASE`

Reply:
done https://github.com/dotCMS/core/pull/30466/files#diff-fa1ceaa19618a6b2bbc30e24c6f930b4971f417db50babb748c2e2837ba9eb82R239